app/assets/stylesheets/application.scss

```diff
@@ -125,3 +127,18 @@ a:hover {
 .panel #created {
   color: $font-color;
 }
+
+.field label {
+  color: $font-color;
+  font-size: 1em;
+}
+
+.tag {
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d8d8d8;
```

Comment:
Color literals like `#d8d8d8` should only be used in variable declarations; they should be referred to via variable everywhere else.